src/main/java/net/sf/jabref/gui/menus/ChangeEntryTypeMenu.java

```diff
@@ -57,19 +57,31 @@ private void populateChangeEntryTypeMenu(JMenu menu, BasePanel panel) {
             for (EntryType type : EntryTypes.getAllValues(BibDatabaseMode.BIBLATEX)) {
                 menu.add(new ChangeTypeAction(type, panel));
             }
+
+            List<EntryType> customTypes = EntryTypes.getAllCustomTypes(BibDatabaseMode.BIBLATEX);
```

Comment:
Would it be possible to extract the following code in a private method? It is duplicated below and the only difference is the mode.